CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-## Unreleased
+## 6.0.0 - 2022-12-22
```

Comment:
Oops 2023 😅

Reply:
Oops, that's why we only merge with reviews. I'll update it, there's also some minor things in the `README.md` still referencing version 5, I'll create a PR for updating this.

Reply:
No problem, @matsales28,  and congratulations for the release! Nice job!

Reply:
Actually, do we need a new version for things like that?